.github/workflows/main.yml

```diff
@@ -3,10 +3,10 @@ name: Claude Code Review
 on:
   pull_request:
     types: [opened, edited]
-  pull_request_review_comment:
-    types: [created, edited]
```

Comment:
The changes remove the triggers for pull request review comments and issue comments. This may reduce the workflow's responsiveness to certain interactions, potentially limiting its functionality.
          ```yaml
            Consider keeping these triggers if they are still needed for the workflow's intended purpose. If they are no longer required, it's good practice to remove them completely rather than commenting them out.
          ```